plugins/GSdx/GSRendererHW.h

```diff
@@ -29,7 +29,7 @@
 
 class GSRendererHW : public GSRenderer
 {
-private:
+protected:
```

Comment:
don't you expose a lot here just to get m_upscale_multiplier?

Also you are not interested in m_upscale_multiplier but you want to know if (m_upscale_multiplier>1). Why not a protected function protected bool IsUpscaled() {return m_upscale_multiplier>1;)?